### DISCUSSION TITLE
Security Onion Crash -- Error 500 -- No License

### DISCUSSION THREAD

**Discussion by flamusdiu** [Security-Onion-Solutions/securityonion#3929] [Unsupported Versions]
Today I just realized on my Security Onion (v2.3.40) that my instance of it crashed with Error 500s. On the Kibana front dashboard, I was give a JSON response telling me the License was not found or something similar. I didn't grab a screenshot, though I should have. 

I hard rebooted the server since even the console (running VM in Proxmox) was not working either. The forced reboot fixed it. I can try and grab some logs to help look at it more, but I have not notice anything with this before.

--- UPDATE --- 
Looks like it's locking my CPU or some reason. I am using a Dell Precision M4800 with Proxmox and it's the only VM on the box at this time. You can see below the info from Proxmox. 
![image](https://user-images.githubusercontent.com/1879197/115642553-2cf0f280-a2e9-11eb-8487-b4c37516063e.png)

Due to the `so-core` process seemingly locking the CPU, I had trouble getting to the console. This is about as far as I got.
![image](https://user-images.githubusercontent.com/1879197/115642563-31b5a680-a2e9-11eb-9089-7580353c98d5.png)

Grafana shows this, which seems to be about the time it went crazy.
![image](https://user-images.githubusercontent.com/1879197/115642576-38441e00-a2e9-11eb-8b38-1afa68ff9125.png)

Any logs that I can look or do I need to find a different method to install it?

**Comment by weslambert** [comment]
With regard to the 500s, are there any clues in the Kibana/ES logs in /opt/so/log?  With regard to CPU, how much traffic are you monitoring? Do you have all services enabled?

**Reply by flamusdiu** [reply]
I monitor probably less then 50 mb/s (though I can go up to 1 gig. I enabled all the services as well. I ended up upping the mem to 15G. I have been tracking it for a day. I think this might have fix it but I'll let you know after the weekend. The config on the site says  12G min but I think that is a bit short for mem.

**Reply by flamusdiu** [reply]
@weslambert looks like I haven't had any crashes the last few days. There isn't that much traffic hitting it but I needed to add more memory then is shown on the site.

**Reply by dougburks** [reply]
From https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs:

> the minimum specs for running Security Onion 2 are:
>
> 12GB RAM
4 CPU cores
200GB storage
>
>Note
>
>These minimum specs are for EVAL mode with minimal services running. These requirements may increase drastically as you enable more services, monitor more traffic, and consume more logs. For more information, please see the detailed sections below.

**Reply by flamusdiu** [reply]
Yes, I am aware. I actually had those specs but they were just a bit under then I required. I just didn't expect the odd problems from having low memory. 

Thank you for the responses and information!
